### PR TITLE
Added profile sshiv5768

### PR DESCRIPTION
**Pull request by sshiv5768**
## Changes proposed

- [x] Added new ``sshiv5768.json`` file.
- [x] Added necessary details like ``name``, ``bio``, ``avatar``.
- [x] Added social media handles. 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
LinkFreeprofile

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1131"></a>

### CODE REVIEW

**Review by KendallDoesCoding — APPROVED** [review]
LGTM ✔️

File: public/data/sshiv5768.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+            "url": "https://github.com/sshiv5768",
+            "icon": "github"
+          },
+          {
+            "name": "Follow me on Twitter",
+            "url": "https://twitter.com/Shivlal01596340",
+            "icon": "twitter"
+          }
+    ]   
+}
```
Comment (Panquesito7) [inline public/data/sshiv5768.json:18]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by kabir0x17 — APPROVED** [review]
Looks good to me 👍

**Review by ssensalo — APPROVED** [review]

**Review by krshdev — APPROVED** [review]

**Review by vikasganiga05 — CHANGES_REQUESTED** [review]
Your JSON file is not formatted correctly. Also missing endline. You can use an online JSON formatter or vscode prettier extension to format correctly.

File: public/data/sshiv5768.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+            "name": "Follow me on GitHub",
+            "url": "https://github.com/sshiv5768",
+            "icon": "github"
+          },
+          {
+            "name": "Follow me on Twitter",
+            "url": "https://twitter.com/Shivlal01596340",
+            "icon": "twitter"
+          }
+    ]
```
[the remark this answers is not in the document]
Comment (Panquesito7) [reply public/data/sshiv5768.json:17]: Formatting will be done automatically by GitHub Actions.

**Review by Pradumnasaraf — APPROVED** [review]
LGTM!

**Review by emmalearnscode — APPROVED** [review]

**Pull request merged**